### ISSUE TITLE
Time to deprecate it

### ISSUE THREAD

**Issue by xHyroM** [Mateo-tem/discord-modals#90]
djs v13.7 is here

**Comment by uSebazz** [comment]
> djs v13.7 is here

I do not think it is necessary to make it obsolete, there may be people who have not updated or do not know how to do it. but will take into consideration **<3**

**Comment by xWass** [comment]
I personally don't think you should deprecate it. I would just leave it as an alternative to using discord.js's own modal system, since this one is really well constructed and easy to use. Discord.js's documention could possibly overcomplicate it and the documentation here is super simple and user friendly

**Comment by volkanceliloglu** [comment]
selectmenu not working on mobile

**Comment by idk-pixel** [comment]
its not time to deprecate, DJS does not support select menus

**Comment by Mateo-tem** [comment]
Moved to #92, there is a poll. If you want, comment there. :)
